Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -101,6 +100,7 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Logging;
 using Prometheus.DotNetRuntime;
+using JsonSerializer = System.Text.Json.JsonSerializer;
```

Comment:
Why are these needed?

Reply:
When I moved this file to the new Serializer, the old one still existed in a namespace which was used in this file.
I'll remove it